PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -317,6 +324,7 @@ def nanoAOD_customizeCommon(process):
         nanoAOD_addDeepBTag_switch = cms.untracked.bool(False),
         nanoAOD_addDeepBoostedJet_switch = cms.untracked.bool(True),
         nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(True),
+        nanoAOD_addDeepDoubleXV2_switch = cms.untracked.bool(True),
```

Comment:
few lines below need to reset as well the nanoAOD_addDeepDoubleXV2_switch 

 ```
  run2_miniAOD_80XLegacy.toModify(nanoAOD_addDeepInfoAK8_switch,
                                    nanoAOD_addDeepBTag_switch = True,
                                    nanoAOD_addDeepBoostedJet_switch = False,
                                    nanoAOD_addDeepDoubleX_switch = False,
                                    **nanoAOD_addDeepDoubleXV2_switch = False,**
                                    nanoAOD_addParticleNet_switch = False,
                                    jecPayload = 'AK8PFchs')
```

Reply:
for run2_nanoAOD_106Xv1  something is done differently than in master:
we are re-running the addDeepDoubleX (and addDeepBoostedJet), any reason for not taking directly from miniAOD like in master ?

I think here we miss this block  like this
 run2_nanoAOD_106Xv1.toModify(
       nanoAOD_addDeepInfoAK8_switch,
       nanoAOD_addDeepBoostedJet_switch = False,
       nanoAOD_addDeepDoubleX_switch = False,
    )